client/components/Layout/AudioControls.vue

```diff
@@ -213,31 +213,26 @@ export default Vue.extend({
       'getPreviousItem',
       'getNextItem'
     ]),
+    ...mapState('playbackManager', ['status', 'repeatMode', 'isShuffling']),
     isPaused(): boolean {
-      return this.$store.state.playbackManager.status === PlaybackStatus.paused;
+      return this.status === PlaybackStatus.paused;
     },
     isPlaying(): boolean {
-      return (
-        this.$store.state.playbackManager.status !== PlaybackStatus.stopped
-      );
+      return this.status !== PlaybackStatus.stopped;
     },
     isRepeating(): boolean {
-      return (
-        this.$store.state.playbackManager.repeatMode !== RepeatMode.RepeatNone
-      );
+      return this.repeatMode !== RepeatMode.RepeatNone;
     },
     repeatIcon(): string {
-      if (
-        this.$store.state.playbackManager.repeatMode === RepeatMode.RepeatOne
-      ) {
+      if (this.repeatMode === RepeatMode.RepeatOne) {
         return 'mdi-repeat-once';
       }
 
       return 'mdi-repeat';
     },
-    isShuffling(): boolean {
-      return this.$store.state.playbackManager.isShuffling;
-    },
+    // TODO: Checking for route is faster to switch the controls than checking for the store
+    // Remove this as soon as we can use the fullpage layout in music pplayer
```

Comment:
```suggestion
    // TODO: Remove this as soon as we can use the fullpage layout in music pplayer
```

Makes it appear in todolists, so it's easier to fix in the future :)